uber/site_sections/summary.py

```diff
@@ -392,3 +392,35 @@ def volunteers_owed_refunds(self, session):
                                                           and a.weighted_hours >= c.HOURS_FOR_REFUND]
             )]
         }
+
+    @csv_file
+    def attendee_birthday_calendar(self, out, session, year=datetime.now(UTC).year):
+        out.writerow(['Subject', 'Start Date', 'Start Time', 'End Date', 'End Time', 'All Day Event', 'Description',
+                      'Location', 'Private'])
+        for person in session.query(Attendee).filter(Attendee.birthdate != None).all():
+            subject = "%s's Birthday" % person.full_name
+            bday = person.birthdate
+            start_date = bday.replace(year=year)
```

Comment:
This line will eventually blow up on you:
```python
>>> import dateparser
>>> dateparser.parse('February 29, 1996').replace(year=2017)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: day is out of range for month
```

Manipulating dates can be quite tricky. What you want to do is calculate the delta in years, and then add that delta to the birthday. This is an arduous process using the standard `datetime` library. I recommend using [dateutil](https://dateutil.readthedocs.io):
```python
>>> from dateutil.relativedelta import relativedelta
>>> birthday = dateparser.parse('February 29, 1996')
>>> delta_years = datetime.date.today().year - birthday.year
>>> birthday + relativedelta(years=delta_years)
datetime.datetime(2017, 2, 28, 0, 0)
```

Our deployment includes `dateparser`, which depends on `dateutil`, so you *can* use it without modifying our `requirements.txt`. However, if you are going to import `dateutil` directly, it's good practice to include it in `requirements.txt`, in case `dateparser` ever changes their internal dependencies:
```
python-dateutil>=2.6.0
```